src/components/MyProfileActivityList/index.js

```diff
@@ -169,11 +183,11 @@ class MyProfileActivityList extends Component {
                   to={`/activity/${Membership.SessionCode}/${Membership.ActivityCode}`}
                   style={membershipItemStyle}
                 >
-                  <img src={Membership.ActivityImagePath} alt="" style={imgStyle} />
+                  <img src={Membership.ActivityImagePath} alt="" className="active" />
```

Comment:
The "className="active" creates a hover effect on the image since it's a link to an activity. This is not seen on the image tag on line 190 because the image is not link to an activity. Therefore, no hover effects should show.